addons/context_actions/functions/fnc_getVehicleWeaponActions.sqf

```diff
@@ -0,0 +1,104 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Dynamically creates child actions to select weapon for turrets with multiple weapons/muzzles/magazines.
+ *
+ * Arguments:
+ * 0: None <OBJECT>
+ * 1: None <OBJECT>
+ * 2: None <OBJECT>
+ * 3: None <OBJECT>
+ * 4: None <OBJECT>
```

Comment:
What are these for? And the given example is wrong.

Reply:
Ah, these params are from CfgContext insertChildren. I'll make a note and fix the example

Reply:
2a19a4f